packages/amplify-category-api/src/__tests__/apigw-walkthrough.test.js

```diff
@@ -1,6 +0,0 @@
-const { getIAMPolicies } = require('../provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough');
-
-test('getIAMPolicies', () => {
-  output = getIAMPolicies('resourceName', ['read']);
-  expect(output.attributes).toStrictEqual(['ApiName', 'ApiId']);
```

Comment:
Did we move this test to some place else now?

Reply:
yes, I didn't remove any tests, I just moved them into a folder structure within __tests__ that mirrors their location in src